src/components/NewAccountPage/NewAccountPage.jsx

```diff
@@ -178,8 +229,8 @@ export default () => {
                   Passwords must match.{' '}
                </StyledFeedback>
             </Form.Group>
-         </Form.Row>
-         <SubmitButton type="submit">Create Account</SubmitButton>
+         </Form.Row> */}
+         <SubmitButton type="submit">Register</SubmitButton>
```

Comment:
This button is improperly placed when I run the app on my phone.
![IMG_0407](https://user-images.githubusercontent.com/46923410/114448271-2eebe080-9b88-11eb-97e6-8ad9bc6a5099.PNG)